internal/cmdopts/cmdoptions.go

```diff
@@ -181,7 +181,7 @@ func (c *Options) NeedsSchemaUpgrade() (upgrade bool, err error) {
 
 // InitWebUI initializes the web UI server
 func (c *Options) InitWebUI(fs fs.FS, logger log.LoggerIface) error {
-	if c.WebUI.WebAddr == "" {
+	if c.WebUI.WebAddr == "" || c.WebUI.WebAddr == "''" {
```

Comment:
It's getting uglier and uglier :-D 

Probably I should add `--web-disable` command-line option

Reply:
It was the most my meager GO skills permitted 😁 Feel free to make it better.

Reply:
It's not about you, it's about my laziness. :) I thought we can mimic behavior without adding new cmdopt. I was wrong

Reply:
Better 😁 Let me integrate it to this PR.

Reply:
Already did